README.md

```diff
@@ -67,67 +73,20 @@ The following foreign languages are supported:
-- [Node.js](https://nodejs.org/en/download/)
-- [NuGet](https://www.nuget.org/downloads/)
-
-| Build command         | Action                                                            |
-|:----------------------|:------------------------------------------------------------------|
-| `make`<sup>[3]</sup>  | Builds `uno` and core library. Works on all platforms.            |
-
- 1. On Windows, we need [vswhere] to locate your Visual Studio 2017 installation. Please make sure we can find `vswhere` in
-    `%PATH%` or at `%PROGRAMFILES(x86)%\Microsoft Visual Studio\Installer`.
- 2. Our cross-platform build scripts are written in `bash`, and `make` is a convenient way to invoke the different build tasks.
- 3. You also can run `bash scripts/build.sh` directly if you don't have `make`.
-
-[vswhere]: https://github.com/Microsoft/vswhere
-
-### Additional make targets
-
-| Build command   | Action                                                                  |
-|:----------------|:------------------------------------------------------------------------|
-| `make install`  | Creates a symlink for `uno` in `/usr/local/bin`.                        |
-| `make release`  | Prepares a release directory for distribution.                          |
-| `make unocore`  | Generates C# code for Uno.Runtime.Core.dll, based on Uno code.          |
-| `make clean`    | Removes build artifacts from the repository and `Packages.SourcePaths`. |
-| `make check`    | Runs the local test suite.                                              |
-
-### Building from IDE
-
-Open `uno.sln` in a capable C# 6.0 IDE, such as Visual Studio 2015+ or JetBrains Rider. Dependencies will be automatically
-installed when building, so you don't really need to use the command-line.
-
-When debugging, make sure to use `main/Uno.CLI.Main` as startup project to launch `uno`'s entrypoint.
-
-### Building the standard library
-
-To build uno's standard library, run `<uno-root>/bin/uno doctor` – or pass `doctor` as parameter to the
-`main/Uno.CLI.Main` project when running from an IDE.
-
-## Running
-
-To run `uno` in your terminal, type `<uno-root>/bin/uno`.
-
-On Windows, some additional runtimes might be needed.
-* VCRedist 2010: [x86](https://www.microsoft.com/en-us/download/details.aspx?id=5555), [x64](https://www.microsoft.com/en-US/Download/confirmation.aspx?id=14632)
-* [VCRedist 2013](https://www.microsoft.com/en-gb/download/details.aspx?id=40784)
-
 ## Contributing
 
-Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of
+Please read [CONTRIBUTING](CONTRIBUTING.md) for details on our code of
 conduct, and the process for submitting pull requests to us.
 
 ### Reporting issues
 
 Please report issues [here](https://github.com/fuse-open/uno/issues).
 
+## Build instructions
+
+Please read [the build instructions](docs/build-instructions.md) for details
+on how to build the source code.
+
```

Comment:
Structually, I would think it's better to put this where the old "## Building"-header was. It's kinda more important information than contributing, I think...

Reply:
I agree and have moved the section.